spec/features/user_can_add_items_to_cart_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rails_helper"
+
+feature "cart" do
+  fixtures :categories
+  fixtures :items
+  fixtures :users
+
+  before(:each) do
+    visit "/"
+  end
+
+  it "can visit the cart page" do
+    click_link("Cart")
+    expect(current_path).to eq('/cart')
+  end
+
+  it "can add items" do
+    click_link("All Products")
+    first(:button, "Add To Cart").click
+    expect(current_path).to eq('/cart')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.